index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Blog</title>
+  <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+  <!--Block Contact-->
+  <div class="contact" id="contact">
+    <div class="contact-header">
+      <p class="contact-header__title">Contact</p>
+      <p class="contact-header__text">For new projects, ideas, business or anything
+        you'd like to discuss, contact with me!</p>
+    </div>
+
+    <div class="main">
+      <div>
+        <form action="#" method="GET" class="main-form">
+          <label>
+            <input type="text" placeholder="Auskteez Tran" class="main-form__input1" name="Auskteez Tran">
+          </label>
+          <label>
+            <input type="text" placeholder="Your email..." class="main-form__input2" name="Your email...">
+          </label>
+          <label>
+            <input type="number" placeholder="Your phone number..." class="main-form__input3" name="Your phone number...">
+          </label>
+          <label>
+            <input type="text" placeholder="Your messge..."  name="Your messge..." class="main-form__input4" >
+          </label>
+        </form>
+      </div>
+
+      <div class="main-contact">
+        <div>
+          <p class="main-contact__address">ADDRESS</p>
+          <p class="main-contact__line-address">123 City name, Street, City, Country</p>
+        </div>
+
+        <div>
+          <p class="main-contact__email">EMAIL</p>
+          <p class="main-contact__line-email">cahee@domain.com</p>
+        </div>
+
+        <div>
+          <p class="main-contact__phone">PHONE</p>
+          <p class="main-contact__line-phone">+012 345 6789</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="footer">
+      <p class="footr__subtext1">© 2015 Company. All right reserved.</p>
+      <p class="footer__logo">Cahee</p>
+      <p class="footr__subtext2">Made with <span class="red">love</span>  by Auskteez</p>
```

Comment:
Write corrent classes - footer_subtext-second

P.S.: do not forget to rename in CSS :)